app/js/components/labOrderEntry/LabTestFieldSet.jsx

```diff
@@ -1,26 +1,35 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { testsData } from './labData';
 import '../../../css/grid.scss';
 
+
+const formatTestName = (testName) => {
+  const name = testName;
+  const formattedName = name.split(' ')
+    .map(splittedName => splittedName[0].toUpperCase() + splittedName.substr(1).toLowerCase())
+    .join(' ')
+    .trim();
+  return formattedName;
+};
+
```

Comment:
The function(`formatTestName`) is used to ensure that the names of the lab-test are formatted and displayed uniformly.
The lab-tests metadata from the server appears in different formats, these are some of them directly from the server:
- `HEMATOCRIT`
- `Hemoglobin`
- `RED BLOOD CELLS`

This function formats the test names to ensure that the first letter of each word is in upper case while the rest is in lower case, consequently, a lab-test like  `RED BLOOD CELLS` becomes `Red Blood Cells`.

Reply:
@IAMOTZ `<p style={{ textTransform: 'capitalize'}}>test.lowerCase()</p>`

Reply:
when you do
```
text.toLowerCase();
// RED BLOOD CELLS =>red blood cells
// HEAMOGLOBIN ==> heamoglobin
```

and then you can use the transform property to get your uppercased first letters

Reply:
Thanks, @rhenshaw56.
This style rule is really what I needed, learnt something new here 😄 
I added the style to the style file of this component instead of an inline style.

Thanks again!